userguide/sharing.rst

```diff
@@ -1006,135 +1006,142 @@ for more details.
-   | worm                | Controls the writability of files and folders depending on their change time and an adjustable grace period.                               |
-   |                     |                                                                                                                                            |
-   +---------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
-   | xattr_tdb           | Stores Extended Attributes (EAs) in a tdb file so they can be used on filesystems which do not provide support for EAs.                    |
-   |                     |                                                                                                                                            |
-   +---------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
-   | zfs_space           | Correctly calculates ZFS space used by the share, including space used by ZFS snapshots, quotas, and resevations. Enabled by default.      |
-   |                     |                                                                                                                                            |
-   +---------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
-   | zfsacl              | Provide ACL extensions for proper integration with ZFS. Enabled by default.                                                                |
-   |                     |                                                                                                                                            |
-   +---------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
-
-
-.. note:: Be careful when using multiple SMB shares, some with and some
-   without *fruit*. macOS clients negotiate SMB2 AAPL protocol
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | Value                | Description                                                                                                                     |
+   |                      |                                                                                                                                 |
+   +======================+=================================================================================================================================+
+   | acl\_\ :file:`tdb`   | Store NTFS ACLs in a tdb file to enable full mapping of Windows ACLs.                                                           |
```

Comment:
No :file: markup in the name, just for the mention as a file extension in the description.